### PR TITLE
Update locales-pt-BR.xml

### PR DESCRIPTION
**Pull request by pirei**
Hi! I want to collab with this document.

## CSL Locales Pull Request Template

You're about to create a pull request to the CSL locales repository.
If you haven't done so already, see <http://docs.citationstyles.org/en/stable/translating-locale-files.html> for instructions on how to translate CSL locale files, and <https://github.com/citation-style-language/styles/blob/master/CONTRIBUTING.md> on how to submit your changes.
In addition, please fill out the pull request template below.

### Description

Briefly describe the changes you're proposing.

### Checklist

- [x] Check that you're listed as a `<translator>` in the `<info>` block at the beginning of the file.

### CODE REVIEW

**Review by bwiernik — CHANGES_REQUESTED** [review]
Thanks! A few questions.

**Comment by adam3smith** [comment]
OK, still some terms to translate left, but no reason to let the perfect the enemy of the obvious improvement here.

**Comment by guslaet** [comment]
I just opened this issue about this file:

https://github.com/citation-style-language/locales/issues/284

Then I tried to fork it and propose the corrections, but the version it loaded was already corrected, so perhaps you have already seen it. It would be nice if these changes are merged.

File: locales-pt-BR.xml [hunk 1]
```diff
@@ -92,48 +95,48 @@
...
+    <term name="review-book">revisão de livro</term>
     <term name="software">software</term>
-    <term name="song">audio recording</term>
-    <term name="speech">presentation</term>
-    <term name="standard">standard</term>
-    <term name="thesis">thesis</term>
+    <term name="song">gravação de áudio</term>
+    <term name="speech">apresentação</term>
+    <term name="standard">padrão</term>
+    <term name="thesis">Tese</term>
```
[the remark this answers is not in the document]
Comment (victormpalma) [reply locales-pt-BR.xml]: Is it possible to replace 'padrão' by 'norma técnica', which is the best translation for 'standard' in this context?

**Comment by pirei** [comment]
> I just opened this issue about this file:
> 
> 
> 
> https://github.com/citation-style-language/locales/issues/284
> 
> 
> 
> Then I tried to fork it and propose the corrections, but the version it loaded was already corrected, so perhaps you have already seen it. It would be nice if these changes are merged.

I did some improvements but I was unable to find them.

**Pull request merged**